scripts/sql/meta.sql

```diff
@@ -0,0 +1,209 @@
+-- create database
+CREATE DATABASE if not exists milvus_meta CHARACTER SET utf8mb4;
+
+/*
+ create tables script
+
+ Notices:
+    id, tenant_id, is_deleted, created_at, updated_at are 5 common columns for all collections.
+ */
+
+-- collections
+CREATE TABLE if not exists milvus_meta.collections (
+    id     BIGINT NOT NULL AUTO_INCREMENT,
+    tenant_id VARCHAR(128) DEFAULT NULL,
+    collection_id BIGINT NOT NULL,
+    collection_name VARCHAR(128),
+    description VARCHAR(2048) DEFAULT NULL,
+    auto_id BOOL DEFAULT FALSE,
+    shards_num INT,
+    start_position VARCHAR(2048),
+    consistency_level INT,
+    ts BIGINT UNSIGNED DEFAULT 0,
+    is_deleted BOOL DEFAULT FALSE,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP on update current_timestamp,
+    PRIMARY KEY (id),
+    INDEX idx_collection_id_ts (collection_id, ts, is_deleted)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+-- collection aliases
+CREATE TABLE if not exists milvus_meta.collection_aliases (
+    id     BIGINT NOT NULL AUTO_INCREMENT,
+    tenant_id VARCHAR(128) DEFAULT NULL,
+    collection_id BIGINT NOT NULL,
+    collection_alias VARCHAR(128),
+    ts BIGINT UNSIGNED DEFAULT 0,
+    is_deleted BOOL DEFAULT FALSE,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP on update current_timestamp,
+    PRIMARY KEY (id),
+    INDEX idx_tenant_id_collection_id_ts_is_deleted (tenant_id, collection_id, ts, is_deleted)
```

Comment:
Should use `collection_alias` as index?

Reply:
No need here. The query case is to select all aliases of a collection